src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -390,7 +390,8 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
       return true
     }
     // Named ULC to BC Limited require a name request.
-    if (this.getOriginalLegalType === CorpTypeCd.BC_ULC_COMPANY && this.isBcLimited) {
+    if (this.getOriginalLegalType === CorpTypeCd.BC_ULC_COMPANY &&
```

Comment:
The old and new code is different from what's in main branch right now. (The line numbers are also the same -- 2 more lines of code in this branch.) I'd like to understand why.

https://github.com/bcgov/business-edit-ui/blob/201cd7691b1b8a0b95e8288cf0f982ea99b91f93/src/components/common/YourCompany/ChangeBusinessType.vue#L391

Reply:
This was also a part of your PR Sev: https://github.com/bcgov/business-edit-ui/commit/47b18fc6c851757daef4fb4b61470a07e761e06b

This is what is looks like in the feature branch:
https://github.com/bcgov/business-edit-ui/blob/2b4793d57065fbd25ac72e519a06a128550acb42/src/components/common/YourCompany/ChangeBusinessType.vue#L393

So, I made a manual resolution here that basically combines both.

Reply:
OK. Thanks for your patience with me! Resolving.

Reply:
Thank you for taking the time to have a close look Sev! ❤️